Andriy_Povzun/HW6_Andriy_Povzun/task3.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+with open('text.txt', 'r') as file:
+    text = file.read()
+upper_alphabet = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K',
+                  'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z']
+lover_aplhabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k',
+                  'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
+
```

Comment:
after lower you can use :
import string
alphabet = string.ascii_lowercase

Reply:
Done. I change all my resolve this task. Check, please.